contract_show_recurring_invoice/views/contract_view.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<openerp>
+    <data>
+
+        <record id="act_recurring_invoices" model="ir.actions.act_window">
+            <field name="context">{'search_default_contract_id':
+                [active_id], 'default_contract_id': active_id}</field>
+            <field name="name">Invoices</field>
+            <field name="res_model">account.invoice</field>
+            <field name="domain">[('contract_id','=',active_id)]</field>
+            <field name="view_id" ref="account.invoice_tree" />
+            <field name="search_view_id" ref="account.view_account_invoice_filter" />
+        </record>
+
+        <record id="account_analytic_account_button_recurring_invoice" model="ir.ui.view">
+            <field name="name">account.analytic.account.button.invoice
+            </field>
+            <field name="model">account.analytic.account</field>
+            <field name="inherit_id" ref="analytic.view_account_analytic_account_form" />
+            <field name="arch" type="xml">
+                <xpath expr='//div[@name="buttons"]' position='inside'>
+                    <button class="oe_stat_button" type="action" icon="fa-edit"
```

Comment:
OK, I understood you bad. I tought you say your button is technical, not the other. Anyway, I would put it next to it, but always visible. If technical features is enabled, you'll see both buttons. If not, you'll only see "Show invoices" button